### ISSUE TITLE
pg must be listed as dependency

### ISSUE THREAD

**Issue by IlyaSemenov** [romeerez/orchid-orm#99]
### Problem

`pg` is not listed as a dependency of `pqb`, but internally `pqb` depends on it:

https://github.com/romeerez/orchid-orm/blob/cda17fe59378d329ee8a1932e733cea4a39b208b/packages/qb/pqb/src/queryMethods/then.ts#L155

`DatabaseError` was only [exported](https://github.com/brianc/node-postgres/pull/2445) in [pg@8.10](https://github.com/brianc/node-postgres/blob/3039f1da7749dc2611a2dbc2dbddc7ef7611937e/CHANGELOG.md#pg860) so that should be the minimum allowed dependency version.

With older `pg` (as I happened to have, unfortunately) this leads to cryptic messages instead of database errors:

```
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
TypeError: Right-hand side of 'instanceof' is not an object
    at then (file:///.../node_modules/.pnpm/pqb@0.11.29/node_modules/pqb/src/queryMethods/then.ts:161:11)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

### Expected

`pg` is listed either as a peer dependency or a direct dependency of `pqb`, with a proper semver. I am not sure what's best so didn't create a PR.

**Comment by romeerez** [comment]
It's because I thought to support other postgres libraries in the future, but as it is only `pg` for now, it makes sense to add it as a direct dependency.

**Comment by romeerez** [comment]
The version that you've pointed out is 8.6, it was around 2 years ago so it should be fine for users.

But if you have an older version, how specifying the dependency would help, won't they conflict?

**Comment by IlyaSemenov** [comment]
My bad, it's 8.6 indeed. I had a project with ^8.4.2 which was updated from 7.x in Oct 2020. I believe the versions wouldn't conflict due to how pnpm manages deps. Then they would have been deduped to the more recent version on the next pnpm dedupe.

**Comment by romeerez** [comment]
Published a change, now `pqb` has pg ^8.6.0 as a regular dependency.
